cypress/e2e/AY11/acessibilidade.cy.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="cypress" />
+import 'cypress-axe'
+
+describe('ACCESSIBILITY TESTS', () => {
+
+  context('CHECK ACCESSIBLITY IN DARK MODE', () => {
+    beforeEach(() => {
+      cy.visit('/')
+      cy.injectAxe()
+      cy.get('#theme-toggle-button').click()
```

Comment:
Que tal verificar que o _ligth mode_ está ativo antes do clique no botão?

Por exemplo: `cy.get('[data-theme="light"]').should('exist')`